Docker gradle setup (#4431)
hashgraph/hedera-services: docker/README.md

This folder contains file that are used to start the services as docker containers.

> All docker workflows are defined and gradle tasks. No skript of file in this folder should be called 'by hand'.

Reviewer: ```suggestion
All docker workflows are defined with a corresponding gradle task. No script or file in this folder should be called 'by hand'.
```
